src/phoenix/trace/llama_index/callback.py

```diff
@@ -487,7 +488,15 @@ def _get_response_output(response: Any) -> Iterator[Tuple[str, Any]]:
         else:
             yield OUTPUT_VALUE, json.dumps(message.additional_kwargs, cls=_CustomJSONEncoder)
             yield OUTPUT_MIME_TYPE, MimeType.JSON
-    else:
+    elif isinstance(response, Response):
+        yield OUTPUT_VALUE, response.response or ""
+        yield OUTPUT_MIME_TYPE, MimeType.TEXT
+    elif isinstance(response, StreamingResponse):
+        # We cannot get the output from a streaming response without exhausting
+        # the stream, so we return an empty string.
+        yield OUTPUT_VALUE, ""
+        yield OUTPUT_MIME_TYPE, MimeType.TEXT
```

Comment:
Might make a note in the code of the follow-up. There's a ticket for this?

Reply:
https://github.com/Arize-ai/phoenix/issues/1867

Reply:
addressed in 2ceba120